src/components/LayersPanel/LayersPanel.js

```diff
@@ -39,4 +49,9 @@ const mapStateToProps = state => ({
   isDisabled: selectors.isElementDisabled(state, 'layersPanel'),
 });
 
-export default connect(mapStateToProps)(withTranslation()(LayersPanel));
+export default connect(
+  mapStateToProps,
+  {
+    setLayers: actions.setLayers,
```

Comment:
Could you please add this to the propTypes?

Reply:
Done

Reply:
Done